src/app/admin/_components/shared/FormFieldWrapper.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const FormFieldWrapper = ({ title, children }) => (
+  <div className="mt-2 w-full">
```

Comment:
It's bad practice to add margins to your root tag, usually it should be provided by props, it will be like `<div className={cn('w-full', className)}>`, where cn is` import cn from 'classnames'`

Reply:
Fixed that. Will take this into account next time!